### PR TITLE
[DOCS-4271] Add Azure AD log collection for US3

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Adds info on collecting Azure AD logs for US3

### Motivation

DOCS-4271, PM request.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by steveharrington1** [comment]
Reviewing Guide for Datadog in the Azure portal update, I think the AD log instructions interrupt the flow - the next section about controlling which logs are sent are relevant for resource logs, which are discussed previously. so i think we should adjust the format a bit. So lets reorder things and update the top to look like this

There are three types of logs that can be emitted from Azure to Datadog using the Datadog resource.
Activity Logs [lonk to section]
Resource Logs [link to section]
Azure Active Directory Logs [link to section]

Then:
Activity Logs content
Resource Logs content, including all the stuff about filtering
Azure AD logs content

File: content/en/integrations/guide/azure-portal.md [hunk 1]
```diff
@@ -110,28 +110,49 @@ Optionally, limit metric collection for Azure VMs and App Service Plans using Az
...
+
+Azure Active Directory (Azure AD) logs contain the history of sign-in activity and audit trail of changes made in Azure AD for a particular tenant. To send Azure AD Logs:
+
+1. Navigate to Azure Active Directory in Azure, and select **Diagnostic Settings** in the left navigation bar.
+2. Click **Add diagnostic setting**.
+3. Select the log categories you want to send to Datadog. Datadog recommends sending all categories.
+4. In **Destination details**, select **Send to a partner solution**.
+5. Select a subscription. Select a Datadog resource in the **Destination** dropdown.
+
+All Azure AD logs from the tenant is sent to the Datadog organization linked to the Datadog resource selected. For cases where you have more than one Datadog resource that link subscriptions to the same Datadog organization, it does not matter which Datadog resource is selected. You only need to set this up once for each Azure tenant.
```
Comment (kayayarai) [inline content/en/integrations/guide/azure-portal.md:154]:
```suggestion
All Azure AD logs from the tenant are sent to the Datadog organization linked to the Datadog resource selected. For cases where you have more than one Datadog resource that link subscriptions to the same Datadog organization, it does not matter which Datadog resource is selected. You only need to set this up once for each Azure tenant.
```

File: content/en/integrations/guide/azure-portal.md [hunk 2]
```diff
@@ -110,28 +110,49 @@ Optionally, limit metric collection for Azure VMs and App Service Plans using Az
...
+* Azure resources with `exclude` tags don't send logs to Datadog.
+* If there's a conflict between inclusion and exclusion rules, exclusion takes priority.
 
 For example, the screenshot below shows a tag rule where only virtual machines, virtual machine scale sets, and app service plans tagged with `Datadog = True` send metrics to Datadog. Resources (of all types) tagged with `Datadog = True` send logs to Datadog.
 
 {{< img src="integrations/guide/azure_portal/metrics-and-logs-tag-rules.png" alt="A screenshot showing a metric tag rule of Datadog=true set for virtual machines, virtual machine scale sets, and app service plans. The logs section is also configured with tag rule of Datadog=true" responsive="true" style="width:100%;">}}
 
+##### Azure Active Directory logs
+
+Azure Active Directory (Azure AD) logs contain the history of sign-in activity and audit trail of changes made in Azure AD for a particular tenant. To send Azure AD Logs:
```
Comment (kayayarai) [inline content/en/integrations/guide/azure-portal.md:146]:
```suggestion
Azure Active Directory (Azure AD) logs contain the history of sign-in activity and an audit trail of changes made in Azure AD for a particular tenant. To send Azure AD Logs:
```

**Review by kayayarai — APPROVED** [review]
Minor grammar fixes and 👍

**Pull request merged**